lib/ecto/migration.ex

```diff
@@ -815,6 +815,8 @@ defmodule Ecto.Migration do
       # Config the Repo (PostgreSQL example)
       config MyApp.Repo, migration_lock: :pg_advisory_lock
 
+  and
```

Comment:
Just trying to separate these code blocks; looks like the config is in the migration file in the current version.

Reply:
Another way would be to used code fences if you want to separate block without using a standalone "and" word. Food for thought.

Reply:
Hmm. I tried that instead, but it's more of a divergence from the style of the rest of the examples. Leaving it like this unless someone objects.

Reply:
I like the way we do it for union et al: https://hexdocs.pm/ecto/Ecto.Query.html#union/2-keywords-examples

What about something like this

```markdown
### Config file
    
    config ...

### Migration File

    defmodule ...

```